### PR TITLE
Remove assertion from test_get_feed_descending

### PR DESCRIPTION
**Pull request by mojito317**
## Checklist

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](../blob/master/DCO1.1.txt)
- [x] Added tests for code changes _or_ test/build only changes
- [x] Updated the change log file (`CHANGES.md`) _or_ test/build only changes
- [x] Completed the PR template below:

## Description

This assertion removal fixes failed pipeline. Comparing seq lengths is not grounded in any contract of the API.

## Approach

I removed the assertion regarding `seq` length comparison.

## Schema & API Changes

- "No change"

## Security and Privacy

- "No change"

## Testing

- Modified existing tests because comparing seq lengths is not grounded in any contract of the API.

## Monitoring and Logging

- "No change"

### CODE REVIEW

File: tests/unit/changes_tests.py [hunk 1]
```diff
@@ -245,7 +245,6 @@ def test_get_feed_descending(self):
                         self.assertTrue(current < last)
                     except AssertionError:
                         self.assertEqual(current, last)
-                        self.assertTrue(len(change['seq']) > len(last_seq))
```
Comment (ricellis) [inline tests/unit/changes_tests.py:248]: Please can you update the test comment block above to remove the reference to this assertion too.

**Review by ricellis — APPROVED** [review]
+1 but we should remove the text talking about the check that is being removed.

File: tests/unit/changes_tests.py (hunk 1 above)
Comment (mojito317) [reply tests/unit/changes_tests.py:248]: Done in 8d968f4

**Review by eiri — APPROVED** [review]
🍏

**Comment by mojito317** [comment]
Thanks for the reviews @ricellis and @eiri! I am about to merge this!

**Pull request merged**